packages/webapp/src/inductions/components/induction-step-endorsement.tsx

```diff
@@ -151,15 +152,29 @@ export const InductionStepEndorsement = (props: Props) => {
             </span>
         );
 
+    const getInductionJourneyRole = () => {
+        if (!props.isCommunityActive) {
+            return InductionRole.GENESIS;
+        } else if (!ualAccount || isInvitee) {
+            return InductionRole.INVITEE;
+        }
+        return InductionRole.INVITER;
+    };
+
+    const getInductionJourneyStep = () => {
+        if (!props.isCommunityActive) {
+            return 2;
+        } else if (isFullyEndorsed) {
+            return 4;
+        }
+        return 3;
+    };
```

Comment:
I think if we replace the step number with the actual step title we get a better readability. (Not for this PR though, should be a nice good first issue :) )